torch/distributed/rpc/api.py

```diff
@@ -2,7 +2,7 @@
 from torch.distributed import invoke_remote_builtin, invoke_remote_python_udf
 from torch.distributed import _start_rpc_agent
 from torch.distributed import _destroy_rref_context, _cleanup_python_rpc_handler
-from torch.distributed import WorkerInfo
+from torch.distributed import WorkerInfo, RRef as PyRRef
```

Comment:
Why do we need `as PyRRef` here?

Reply:
I wanted to expose RRef to rpc.RRef  -- but avoid the lint issues for RRef not being used in the file. I can alternatively add a lint ignore rule. Is this how you'd do it?

Reply:
Yep, either adding a `# noqa: F401` or do something similar to this PR #25823